ISISComputingGroup/JSON_bourne
# Ticket 2128: Improve deploy process

See https://github.com/ISISComputingGroup/IBEX/issues/2128

To test:
* Run the deploy batch file
* Confirm that the new file structure matches that on the external webserver
